envbuilder.go

```diff
@@ -112,22 +112,25 @@ func Run(ctx context.Context, opts options.Options) error {
 	var fallbackErr error
 	var cloned bool
 	if opts.GitURL != "" {
-		cloneOpts, err := git.CloneOptionsFromOptions(opts)
-		if err != nil {
-			return fmt.Errorf("git clone options: %w", err)
-		}
-
 		endStage := startStage("📦 Cloning %s to %s...",
 			newColor(color.FgCyan).Sprintf(opts.GitURL),
-			newColor(color.FgCyan).Sprintf(cloneOpts.Path),
+			newColor(color.FgCyan).Sprintf(opts.WorkspaceFolder),
 		)
-
 		stageNum := stageNumber
-		w := git.ProgressWriter(func(line string) { opts.Logger(log.LevelInfo, "#%d: %s", stageNum, line) })
+		logStage := func(format string, args ...any) {
+			opts.Logger(log.LevelInfo, "#%d: %s", stageNum, fmt.Sprintf(format, args...))
+		}
+
+		cloneOpts, err := git.CloneOptionsFromOptions(logStage, opts)
+		if err != nil {
+			return fmt.Errorf("git clone options: %w", err)
+		}
+
+		w := git.ProgressWriter(func(line string) { logStage(line) })
```

Comment:
Maybe we just change the progress writer signature to match, saves us from the function wrapping, wdyt?

Reply:
Yep just realised that! Done. Also forgot that `%` in git output will end up `%(MISSING)` 😅